lms/static/js/onboarding_survey/onboarding_survey.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function() {
+
+
+    function hide_or_show_org_email_field(){
+
+            reg_update_poc_radio = $('input[name=is_poc]:checked').val();
+
+            if (reg_update_poc_radio == '1'){
+                $("label[for='id_org_admin_email']").hide();
```

Comment:
we should hid the main container of `org_admin_email` filed instead of hiding each child.

Reply:
The parent is a barebone html tag with no id or class. So, had to revert to this.